modules/ui/fields/input.js

```diff
@@ -322,6 +322,7 @@ export function uiFieldText(field, context) {
             .attr('title', isMixed ? tags[field.key].filter(Boolean).join('\n') : undefined)
             .attr('placeholder', isMixed ? t('inspector.multiple_values') : (field.placeholder() || t('inspector.unknown')))
             .classed('mixed', isMixed);
+        updatePhonePlaceholder();
```

Comment:
The other call to this function is conditional on the field’s type:

https://github.com/openstreetmap/iD/blob/43a9d275498d2dd1530e4ed7e9e1c90a272a380b/modules/ui/fields/input.js#L95

Should the conditional be repeated here?

Reply:
good spotting thanks, I lost a commit somehow......